modules/nosql/manager_redis.go

```diff
@@ -59,8 +60,84 @@ func (m *Manager) GetRedisClient(connection string) redis.UniversalClient {
 		name: []string{connection, uri.String()},
 	}
 
+	opts := getRedisOptions(uri)
+	tlsConfig := getRedisTlsOptions(uri)
```

Comment:
The mega switch statement that was here originally has been broken into these two distinct units of work.

Reply:
TBH the reason why I wrote the big switch statement in the first place is because as you've seen I don't think it gains much and it's still a megaswitch statement.

But it's fine.

Reply:
Oh yeah sorry! Didn't mean to offend, just meant that I had moved the block to make it practical for testing without instantiating the full redis client.